diff --git a/node_modules/expo-speech-recognition/android/src/main/java/expo/modules/speechrecognition/ExpoSpeechRecognitionOptions.kt b/node_modules/expo-speech-recognition/android/src/main/java/expo/modules/speechrecognition/ExpoSpeechRecognitionOptions.kt
index 6832c29..e8b8064 100644
--- a/node_modules/expo-speech-recognition/android/src/main/java/expo/modules/speechrecognition/ExpoSpeechRecognitionOptions.kt
+++ b/node_modules/expo-speech-recognition/android/src/main/java/expo/modules/speechrecognition/ExpoSpeechRecognitionOptions.kt
@@ -4,9 +4,7 @@ import android.media.AudioFormat
 import android.speech.RecognizerIntent
 import expo.modules.kotlin.records.Field
 import expo.modules.kotlin.records.Record
-import expo.modules.kotlin.types.OptimizedRecord
 
-@OptimizedRecord
 class SpeechRecognitionOptions : Record {
     @Field
     val interimResults: Boolean? = false
@@ -60,7 +58,6 @@ class SpeechRecognitionOptions : Record {
     val iosVoiceProcessingEnabled: Boolean? = false
 }
 
-@OptimizedRecord
 class VolumeChangeEventOptions : Record {
     @Field
     val enabled: Boolean? = false
@@ -69,7 +66,6 @@ class VolumeChangeEventOptions : Record {
     val intervalMillis: Int? = null
 }
 
-@OptimizedRecord
 class RecordingOptions : Record {
     @Field
     val persist: Boolean = false
@@ -87,7 +83,6 @@ class RecordingOptions : Record {
     val outputEncoding: String? = null
 }
 
-@OptimizedRecord
 class AudioSourceOptions : Record {
     @Field
     val uri: String = ""
@@ -105,13 +100,11 @@ class AudioSourceOptions : Record {
     val chunkDelayMillis: Long? = null
 }
 
-@OptimizedRecord
 class GetSupportedLocaleOptions : Record {
     @Field
     val androidRecognitionServicePackage: String? = null
 }
 
-@OptimizedRecord
 class TriggerOfflineModelDownloadOptions : Record {
     @Field
     val locale: String = "en-US"
